### DISCUSSION TITLE
Group Android notifications?

### DISCUSSION THREAD

**Discussion by mirenradia** [community/community#60056] [Mobile]
### Select Topic Area

Product Feedback

### Body

With most other Android apps, multiple notifications from the same app are grouped (See [Android developer documentation](https://developer.android.com/develop/ui/views/notifications/group)). For some reason, with the GitHub Mobile app, notifications are often not grouped (in particular failed or cancelled workflow runs) which can be quite annoying when I want to dismiss all of them (but also don't want to turn notifications off). See the screenshot below as an example:
![Many ungrouped notifications from GitHub Mobile on Android](https://github.com/community/community/assets/32646026/d3f00c1f-6398-4541-889d-104243053527)

Would it be possible to group these notifications?

I have the following phone in case that's important:
* Google Pixel 7 Pro
* Android 13
* Android security update: 5 June 2023
* Build number TQ3A.230605.012

**Comment by alcere** [comment]
Hi @mirenradia! Thanks for your feedback! I think you are describing a valid use case. I will create an internal issue to discuss and explore your proposal!

**Reply by Th3S4mur41** [reply]
Weird thing is, I'm pretty sure grouping worked in the past...
The behavior changed a few weeks/months ago for me 🤔

**Comment by johnsmith3321** [comment]
An amazing message and call notification-controlling application developed for Android is BuzzKill.

You can download the Premium version without any cost from [Here](https://blackapks.com/en/buzzkill)
